Cargo.toml

```diff
@@ -38,6 +38,7 @@ default-members = [
 
 [package.metadata.docs.rs]
 all-features = true
+rustc-args = ["--cfg", "docsrs"]
```

Comment:
why do we need this?

Reply:
cfg is a rustc arg not a rustdoc

https://docs.rs/about/builds#cross-compiling